### ISSUE TITLE
Cant install Breadboard on Win10 Machines

### ISSUE THREAD

**Issue by JuKo007** [human-nature-lab/breadboard#17]
I´m running Windows 10.0.1/134.48 and I cannot install breadboard on three different machines because I always get the same error. Upon running the command line, I get:

`Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
Picked up _JAVA_OPTIONS: "--add-modules=java.xml.bind"
Unrecognized option: --add-modules=java.xml.bind
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.`

**Comment by disperse** [comment]
Yes, this is an error with the breadboard.bat file that is included with breadboard v2.3.0. We'll correct this in v2.3.1. In the meantime, replace the contents of your breadboard.bat file with the following:
```
@ECHO OFF &SETLOCAL
for /f tokens^=2-5^ delims^=.-_^" %%j in ('java -fullversion 2^>^&1') do set "jver=%%j%%k%%l%%m"
set "n=0123456789%jver%"
set "n=%n:~-10,1%"
set x=10
set j9=1
set /a n=n - 2
for /L %%i in (1,1,%n%) do set /a j9*=x
set /a j9=j9*19
rem Set java options for jdk9 and above
if %jver% GTR %j9% (set _JAVA_OPTIONS="--add-modules=java.xml.bind")
rem run the breadboard server
java -cp "./lib/*;" -Dhttps.port=9443 -Dapplication.secret="pleasechangethis" -Dconfig.file=conf\application-prod.conf -Xms1024m -Xmx1024m -XX:MaxPermSize=256m -XX:ReservedCodeCacheSize=128m -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC play.core.server.NettyServer
```

**Comment by JuKo007** [comment]
Works for me now! Thanks a lot!

**Comment by Mira217** [comment]
Hi sorry, I have exactly the same issue but your solution did not work for me. Could you please explain how to change the contents in the file? I converted the file to .txt and replaced the text, then converted it back to .bat. However, this time I have this error "'breadboard.bat' is not recognized as an internal or external command, operable program or batch file.". Sorry I am new to this.

Thanks

**Comment by alhk** [comment]
> Hi sorry, I have exactly the same issue but your solution did not work for me. Could you please explain how to change the contents in the file? I converted the file to .txt and replaced the text, then converted it back to .bat. However, this time I have this error "'breadboard.bat' is not recognized as an internal or external command, operable program or batch file.". Sorry I am new to this.
> 
> Thanks

Don't put it in a .txt, just execute the text from disperse directly in your command prompt. I had this issue too, found this, and the code worked for me as well.
